conftest.py

```diff
@@ -455,6 +458,57 @@ def service_actions():
         httpd.serve_forever()
 
 
+def run_in_pyodide(*args):
+    # This can be called as @run_in_pyodide or @run_in_pyodide("first_dep",
+    # "second_dep"). In the former case, run_in_pyodide acts as the decorator.
```

Comment:
If it can only be called with the second variant and it allow to simplify that's also fine.